charmander/resources/example_resources.py

```diff
@@ -0,0 +1,43 @@
+"""
+Example resources.
+
+"""
+from marshmallow import Schema, fields
+from microcosm_flask.linking import Link, Links
+from microcosm_flask.namespaces import Namespace
+from microcosm_flask.operations import Operation
+from microcosm_flask.paging import PageSchema
+
+from charmander.models.example_model import Example
+
+
+class NewExampleSchema(Schema):
```

Comment:
We use `marshmallow` for building the schemas for our resources which then allows us to auto-generate `swagger` schemas for our services. This lets us auto-generate service clients.